test/lib/code_corps/messages/messages_test.exs

```diff
@@ -292,4 +299,28 @@ defmodule CodeCorps.MessagesTest do
       assert result.id == conversation.id
     end
   end
+
+  describe "get_part/1" do
+    test "gets a single part" do
+      conversation_part = insert(:conversation_part)
+
+      result = Messages.get_part(conversation_part.id)
+
+      assert result.id == conversation_part.id
+    end
+  end
+
+  describe "add_part/1" do
+    test "creates a conversation part" do
+      conversation = insert(:conversation)
+      user = insert(:user)
+      attrs = %{
+        author_id: user.id,
+        body: "Test body",
+        conversation_id: conversation.id
+      }
+
+      {:ok, %ConversationPart{}} = Messages.add_part(attrs)
```

Comment:
Should at least assert it belongs to the conversation here, if not checking the rest of the attributes.